packages/microcosm-www-next/src/stylesheets/components/_browser-graphic.scss

```diff
@@ -44,6 +44,36 @@ $left-padding-lg: rem(40);
     @include breakpoint(small-tablet-up) {
       height: calc(100% - #{$header-height-lg});
     }
+
+    #graphic-6 &,
+    #graphic-7 & {
+      position: relative;
+
+      &::before {
+        align-items: center;
+        background: rgba(34, 35, 40, 0.8);
+        content: url('../images/loading.png');
+        display: flex;
+        height: 100%;
+        justify-content: center;
+        position: absolute;
+        width: 100%;
+        z-index: 1;
+      }
+
+      &::after {
+        background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA0AAAANCAYAAABy6+R8AAAAcUlEQVR4AZWSAQbAMAxFB3rB7FrdMbbuiCtkeRTEfq3lh/zkaVrZ3L2EzpCFyJWoX/ST3CFOn4D4z+hrGDuAAhPQyTE1OIDsU1BgVRMQFOhqZELWkaCKP4Ns9SZbfZP8vS9fA0iAeiM02DAK4M/do6+8BvNDPMuaalkAAAAASUVORK5CYII=);
```

Comment:
why the base64 encoded img?

Reply:
I thought it'd be better than requesting another resource, thoughts? This one is a small X 
![screen shot 2018-02-06 at 4 51 42 pm](https://user-images.githubusercontent.com/22921416/35886147-0fa164ba-0b5e-11e8-8e2c-2730cc22f389.png)

Reply:
I don't know, but if it is then sweet. I was mostly just curious